packages/odie-client/src/constants.ts

```diff
@@ -46,6 +46,14 @@ export const ODIE_TRANSFER_MESSAGE: Message[] = [
 	},
 ];
 
+export const ODIE_THIRD_PARTY_MESSAGE = `${ __(
+	'I’m happy to connect you to a human! However, it looks like 3rd party cookies are disabled in your browser. We need you to turn them on for our live chat to work properly. [Use our guide](https://wordpress.com/support/third-party-cookies/)',
```

Comment:
> We need you to turn them on for our live chat to work properly.
This is a little demanding ;) "Please turn them on" maybe?

Reply:
Good call. That works! Let's update it. 

> I’m happy to connect you to a human! However, it looks like 3rd party cookies are disabled in your browser. Please turn them on for our live chat to work properly. [Use our guide](https://wordpress.com/support/third-party-cookies/)'